draft-ietf-quic-multipath.md

```diff
@@ -676,184 +671,8 @@ ACK frames are received in 1-RTT packets while the state of multipath
-"ACK Delay" field is the delay time of the path's largest received packet.
-
-For packet sender (ACK receiver):
-
-* Maintain an unacked list for each path to retrieve the packets that has been
-sent when an ACK is received. It can coexist with the unacked list of the
-connection layer or packet number space layer.
-
-* Generate RTT sample for a path when the following conditions are met: (1) the
-largest acknowledged packet number is newly acknowledged by the ACK received
-from this path, and (2) at least one of the newly acknowledged packets was ack-eliciting.
-
-### ECN and Zero-Length CID Considerations {#ecn-handling}
-
-ECN feedback in QUIC is provided based on counters in the ACK frame
-(see {{Section  19.3.2. of QUIC-TRANSPORT}}). That means if an ACK
-frame acknowledges multiple packets, the ECN feedback cannot be accounted
-to a specific packet.
-
-There are separate counters for each packet number space. However, sending
-to zero-length CID receivers, the same number space is used for multiple paths.
-Respectively, if an ACK frames acknowledges multiple packets from different paths,
-the ECN feedback cannot unambiguously be assigned to a path.
-
-If the sender marks its packets with the ECN capable flags, the network
-will expect standard reactions to ECN marks, such as slowing down
-transmission on the sending path. If zero-length CID is used,
-the sending path is however ambiguous. Therefore, the sender MUST
-treat a CE marking as a congestion signal on all sending paths that
-have been by a packet that was acknowledged in the ACK frame signaling
-the CE counter increase.
-
-A host that is sending over multiple paths to a zero-length CID receiver
-MAY disable ECN marking and
-send all subsequent packets as Not-ECN capable.
-
-### Restricted Sending to Zero-Length CID Peer
-
-Hosts that are designed to support multipath using multiple number spaces
-MAY adopt a conservative posture after negotiating multipath support with
-a peer using zero-length CID. The simplest posture is to only send
-data on one path at a time, while accepting packets on all acceptable
-paths. In that case:
-
-* the attribution of packets to path discussed in {{zero-length-cid-loss-and-congestion}}
-  are easy to solve because packets are sent on a single path,
-* the ACK Delays are correct,
-* the vast majority of ECN marks relate to the current sending path.
-
-Of course, the hosts will only take limited advantage from the multipath
-capability in these scenarios. Support for "make before break" migrations
-will improve, but load sharing between multiple paths will not work.
-
-## Using non-zero length CID and Multiple Packet Number Spaces
-
-If packets contain a non-zero CID, each path has
-its own packet number space for transmitting 1-RTT packets and a new
-ACK frame format is used as specified in {{ack-mp-frame}}.
+A new ACK frame format is used as specified in {{ack-mp-frame}} that
+MUST be used for for acknolowing 1-RTT packets.
```

Comment:
The added text conflicts with the latter one saying "endpoints SHOULD use ACK_MP frames instead of ACK", and I don't think receiving ACK frames to acknowledge path 0 is actually an issue. Maybe this would be better?
```suggestion
The ACK_MP frame, as specified in {{ack-mp-frame}}, is used to
acknowledge 1-RTT packets.
```